modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -188,29 +188,31 @@
 
 /obj/item/gun/ballistic/automatic/pistol/pdh/corpo
 	name = "\improper PDH-6M 'Corpo'"
-	desc = "A prestigious ballistic sidearm, from Armadyne's military division, normally given to corporate agents. It has a 3 round burst mode and uses .357 Magnum ammunition."
+	desc = "A prestigious ballistic sidearm, from Armadyne's military division, normally given to corporate agents. It has a 2 round burst mode and uses .38 Special ammunition."
 	icon_state = "pdh_corpo"
 	w_class = WEIGHT_CLASS_NORMAL
 	mag_type = /obj/item/ammo_box/magazine/multi_sprite/pdh_corpo
 	can_suppress = FALSE
 	fire_sound = 'modular_skyrat/modules/sec_haul/sound/hpistol_fire.ogg'
-	burst_size = 3
-	fire_delay = 2
-	spread = 5
+	burst_size = 2
+	fire_delay = 4
+	spread = 12
 	realistic = TRUE
 	dirt_modifier = 0.1
 	company_flag = COMPANY_ARMADYNE
 
 /obj/item/ammo_box/magazine/multi_sprite/pdh_corpo
 	name = "\improper PDH-6M magazine"
-	desc = "A magazine for Armadyne's exclusive corporate handgun. Chambered for .357, to your disgrace."
+	desc = "A magazine for Armadyne's exclusive corporate handgun. Chambered for .38, to your disgrace."
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/mags.dmi'
 	icon_state = "pdh"
-	ammo_type = /obj/item/ammo_casing/a357
-	caliber = "357"
-	max_ammo = 14
+	ammo_type = /obj/item/ammo_casing/c38
+	caliber = CALIBER_38
```

Comment:
Leave this gun as it was, it is designed to be very powerful.

Reply:
Sure, low-alert PMCs now have the PDH 'Striker', which is the version of the corpo shown in the PR (with a new sprite), PDH Corpo nerfs reverted